### PR TITLE
Embed schedule image/link on updates page

### PR DESCRIPTION
**Pull request by MattIPv4**
## Describe your changes

As an interim solution until #327 is implemented properly with the schedule data living in the DB, this PR adds an embedded image of the schedule short-link that was enabled with the work in #535.

## Notes for testing your change

If the schedule short-link does not exist, a no schedule available message is shown.

If the schedule short-link does exist, the imgur image is embedded via next/image.

### CODE REVIEW

**Comment by MattIPv4** [comment]
I don't have access to shortlinks on preview.alveus.gg to add a link to that DB for testing.

**Comment by pjeweb** [comment]
> I don't have access to shortlinks on preview.alveus.gg to add a link to that DB for testing.

You should have super user perms now

File: apps/website/src/components/notifications/Schedule.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  return (
+    <Link
+      href={`${getShortBaseUrl()}/l/${schedule.data.slug}`}
+      external
+      custom
+      className="group"
+    >
+      <Image
+        src={schedule.data.link}
```
Comment (pjeweb) [inline apps/website/src/components/notifications/Schedule.tsx:23]: Should we do a very basic check if its an imgur image link?

**Review by pjeweb — APPROVED** [review]

File: apps/website/src/components/notifications/Schedule.tsx (hunk 1 above)
Comment (MattIPv4) [reply apps/website/src/components/notifications/Schedule.tsx:23]: I thought about doing that, but decided I trust Mik enough to stick to using imgur, and if not, I kinda want to see it error so we know to fix it.

**Pull request merged**